### DISCUSSION TITLE
Remove duplicate plugin from Jetbrains marketplace

### DISCUSSION THREAD

**Discussion by ChillarAnand** [reloadware/reloadium#101] [General]
A lot of people use the marketplace to download the plugin. However, there is a duplicate plugin for reloadium.

Can we just get it removed?

**Comment by ChillarAnand** [comment]
Just noticed that it is a fork of the plugin.

In that case, they should use a different or modified icon for the plugin. Just to avoid confusion for the new users.

**Reply by dkrystki** [reply]
Yeah it's a fork made by a company not related to this project. I've asked them to change the logo and name.
